esx_service/cli/vmdkops_admin.xml

```diff
@@ -0,0 +1,571 @@
+<?xml version="1.0"?>
+<plugin
+    xmlns="http://www.vmware.com/Products/ESX/6.0/esxcli/">
+    <!-- esxcli extension syntax version -->
+    <version>1.0.0</version>
+    <namespaces>
+        <namespace path="vdvs">
```

Comment:
Yes. Can use main namespace as ```esxcli storage containers <further commands>```.
However, putting everything under volumes also seems to be unsound.
For eg. 

the command to set access to a vmgroup would endup something like:
```esxcli storage containers volumes vmgroup access set <options>```

What do you think?